cli/login.go

```diff
@@ -169,14 +174,21 @@ func LoginCommand(app *kingpin.Application, input LoginCommandInput) {
 		app.Fatalf("Expected a response with SigninToken")
 		return
 	}
-
-	destination := "https://console.aws.amazon.com/"
-	if profile, _ := awsConfig.Profile(input.Profile); profile.Region != "" {
-		destination = fmt.Sprintf(
-			"https://%s.console.aws.amazon.com/console/home?region=%s",
-			profile.Region, profile.Region,
-		)
-	}
+  
+  destination := "https://console.aws.amazon.com/"
+  if input.Service != "" {
+  	destination = fmt.Sprintf(
+  			"https://console.aws.amazon.com/%s",
+  			input.Service,
+  		)
+	  } else {
+  	if profile, _ := awsConfig.Profile(input.Profile); profile.Region != "" {
```

Comment:
So service won't be handled for specific regions?

Reply:
:) I was waiting for this question. 
There are certain endpoints - that are global across all AWS - and there are others that are region specific.

The behavior that currently happens is that the browser will go into the region that you last worked in. (I assume it is a cookie that is used - AWSCRegion or noflush_Region) where the region is stored

Do you prefer having it region specific? I do not think that it is necessary